domain/objectstore/service/service_test.go

```diff
@@ -34,16 +34,18 @@ func (s *serviceSuite) TestGetMetadata(c *gc.C) {
 
 	path := uuid.MustNewUUID().String()
 
-	metadata := coreobjectstore.Metadata{
-		Path: path,
-		Hash: uuid.MustNewUUID().String(),
-		Size: 666,
+	metadata := objectstore.Metadata{
+		Path:        path,
+		Hash256:     uuid.MustNewUUID().String(),
+		Hash512_384: uuid.MustNewUUID().String(),
+		Size:        666,
```

Comment:
nit: UUIDs are not valid hashes.

This may come up in future if we add more validation, but imo we can cross that bridge when we come to it

Reply:
The original intention was to create some random strings to represent a hash. I get what you mean, but essentially we want a random string to test that we're not special casing the hash.